tslint.json

```diff
@@ -5,9 +5,18 @@
     ],
     "jsRules": {},
     "rules": {
-        "interface-name": [true, "never-prefix"],
+        "interface-name": [
+            true,
+            "never-prefix"
+        ],
         "max-classes-per-file": false,
-        "object-literal-sort-keys": false
+        "object-literal-sort-keys": false,
+        "ordered-imports": [
+            true,
+            {
+                "import-sources-order": "any"
```

Comment:
I am not able to separate those by a new line while using sort-imports plugin that was suggested as common "best practices" settings (https://atomist-community.slack.com/files/kipz/F57TXD2QJ/vs_code_user_settings.js), specifically:

```
  "typescript.extension.sortImports.quoteStyle": "double",
  "typescript.extension.sortImports.sortOnSave": true,
  "typescript.extension.sortImports.sortMethod": "path",
```

It rearranges imports in this specific order on save :(